invenio_requests/assets/semantic-ui/js/invenio_requests/timeline/TimelineFeed.js

```diff
@@ -39,15 +40,15 @@ class TimelineFeed extends Component {
   };
 
   render() {
-    const { timeline, loading, error } = this.props;
+    const { timeline, loading, error, setPage, size, page } = this.props;
     const { modalOpen, modalAction } = this.state;
 
     return (
       <Loader isLoading={loading}>
         <Error error={error}>
           <Overridable id="TimelineFeed.layout" {...this.props}>
             <Container>
-              <Segment>
+              <Segment id="request-details-feed-segment">
```

Comment:
```suggestion
              <Segment className="borderless shadowless">
```
see below in CSS for the full explanation

Reply:
sure :+1: